packages/react/src/components/Footer/LegalNav.js

```diff
@@ -76,6 +75,30 @@ function renderListItems(links) {
       </li>
     );
   });
+
+  return renderedLinks.map((elem, index, origin) => {
+    if (index == 0) {
+      return (
+        <ul className={`${prefix}--legal-nav__holder`} key={index}>
+          {elem}
+        </ul>
+      );
+    } else if (index % 2 != 0 && origin[index + 1] != undefined) {
+      return (
+        <ul className={`${prefix}--legal-nav__holder`} key={index}>
+          {elem}
+          {origin[index + 1]}
+        </ul>
+      );
+    } else if (origin[index + 1] != undefined) {
+      return (
+        <ul className={`${prefix}--legal-nav__holder`} key={index}>
+          {origin[index + 1]}
+          {renderTrusteItem()}
```

Comment:
Just trying to check the logic on this, the `renderTrusteItem` always gets rendered in any cases? Since this is a legal thing we really need to make sure that this is always here (and only rendered once).